packages/pieces/approval/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { PieceAuth, createPiece } from "@activepieces/pieces-framework";
+import { approvalAction } from "./lib/actions/approval";
+
+export const approval = createPiece({
+  displayName: "Approval",
+  auth: PieceAuth.None(),
+  logoUrl: "https://cdn.activepieces.com/pieces/approval.png",
+  authors: ['abuaboud'],
```

Comment:
change this

Reply:
done in [d780c9d](https://github.com/activepieces/activepieces/pull/1635/commits/d780c9df4b28c01cc0cc63ebad16ffd73419764a)